RELEASING.md

```diff
@@ -1,6 +1,6 @@
 # Release Checklist
 
-- Update to the new version number using the [Bump Version](https://github.com/IronCoreLabs/cloaked-ai/actions/workflows/bump-version.yaml) workflow. This will be used as the Rust/Python/Kotlin version number.
-- The [Python/Kotlin Release](https://github.com/IronCoreLabs/cloaked-ai/actions/workflows/sdk-release.yaml) action will automatically publish the Python/Kotlin SDKs.
+- Update to the new version number using the [Bump Version](https://github.com/IronCoreLabs/ironcore-alloy/actions/workflows/bump-version.yaml) workflow. This will be used as the Rust/Python/Kotlin version number.
+- The [Python/Kotlin Release](https://github.com/IronCoreLabs/ironcore-alloy/actions/workflows/sdk-release.yaml) action will automatically publish the Python/Kotlin SDKs.
 - When the JVM artifacts have been deployed, go to https://oss.sonatype.org, log in using the `icl-devops` username and
```

Comment:
I think you could link to this from the `RELEASING` doc in the python directory. The info in there is still good to have in case we need it, but it should be clear we expect this automation to be used generally.